bin/install_uninstall.ml

```diff
@@ -397,42 +433,86 @@ let install_uninstall ~what =
-               the same prefix as the ocaml compiler.")
-    and+ libdir_from_command_line =
+               $(i,\\$prefix/lib), etc...")
+    and+ destdir =
       Arg.(
         value
         & opt (some string) None
+        & info [ "destdir" ] ~env:(env_var "DESTDIR") ~docv:"PATH"
+            ~doc:
+              "When passed, this directory is prepended to all installed paths.")
+    and+ libdir_from_command_line =
+      Arg.(
+        value
+        & opt (some absolute_path) None
         & info [ "libdir" ] ~docv:"PATH"
             ~doc:
               "Directory where library files are copied, relative to \
                $(b,prefix) or absolute. If $(b,--prefix) is specified the \
-               default is $(i,\\$prefix/lib), otherwise it is the output of \
-               $(b,ocamlfind printconf destdir)")
-    and+ destdir =
+               default is $(i,\\$prefix/lib). Only absolute path accepted.")
+    and+ mandir_from_command_line =
+      let doc =
+        "When passed, manually override the directory to install man pages. \
+         Only absolute path accepted."
+      in
       Arg.(
         value
-        & opt (some string) None
-        & info [ "destdir" ] ~env:(env_var "DESTDIR") ~docv:"PATH"
-            ~doc:
-              "When passed, this directory is prepended to all installed paths.")
-    and+ mandir =
+        & opt (some absolute_path) None
+        & info [ "mandir" ] ~docv:"PATH" ~doc)
+    and+ docdir_from_command_line =
+      let doc =
+        "When passed, manually override the directory to install \
+         documentation. Only absolute path accepted."
+      in
+      Arg.(
+        value
+        & opt (some absolute_path) None
+        & info [ "docdir" ] ~docv:"PATH" ~doc)
+    and+ etcdir_from_command_line =
       let doc =
-        "When passed, manually override the directory to install man pages"
+        "When passed, manually override the directory to install configuration \
+         files. Only absolute path accepted."
       in
-      Arg.(value & opt (some string) None & info [ "mandir" ] ~docv:"PATH" ~doc)
-    and+ docdir =
+      Arg.(
+        value
+        & opt (some absolute_path) None
+        & info [ "etcdir" ] ~docv:"PATH" ~doc)
+    and+ bindir_from_command_line =
       let doc =
-        "When passed, manually override the directory to install documentation"
+        "When passed, manually override the directory to install public \
+         binaries. Only absolute path accepted."
       in
-      Arg.(value & opt (some string) None & info [ "docdir" ] ~docv:"PATH" ~doc)
-    and+ etcdir =
+      Arg.(
+        value
+        & opt (some absolute_path) None
+        & info [ "bindir" ] ~docv:"PATH" ~doc)
+    and+ sbindir_from_command_line =
       let doc =
-        "When passed, manually override the directory to install configuration \
-         files"
+        "When passed, manually override the directory to files from sbin \
```

Comment:
It's not an issue introduced by this PR, but I find this `when passed,` preamble to be redundant.